universal-application-tool-0.0.1/app/views/admin/ProgramNewOne.java

```diff
@@ -0,0 +1,24 @@
+package views.admin;
+
+import static j2html.TagCreator.*;
+
+import play.mvc.Http.Request;
+import play.twirl.api.Content;
+import views.BaseHtmlView;
+
+public final class ProgramNewOne extends BaseHtmlView {
+
+  public Content render(Request request) {
+    return htmlContent(
+        body(
+            h1("Create a new Program"),
+            div(
+                form(
+                        makeCsrfTokenInputTag(request),
+                        textField("name", "Program Name"),
+                        textField("description", "Program Description"),
+                        submitButton("Create"))
+                    .withMethod("post")
+                    .withAction("/admin/programs"))));
```

Comment:
Suggestion (blocking): use the reverse router to get the URL path

Reply:
done!

Reply:
there isn't actually a `withAction` that takes a `Call`, so I think I have to use `controllers.routes.AdminProgramController.newOne().url()`